tests/phpunit/ConfigDefinitionTest.php

```diff
@@ -86,6 +87,7 @@ public function provideValidConfigs(): array
                                 ['name' => 'dwh3', 'permission' => UserDefinition::PERMISSION_WRITE],
                             ],
                             'disabled' => false,
+                            'reset_password' => false,
                         ],
```

Comment:
Chybí test na to, že tam může ten reset password být - tys ho přidal jen do zparsovaného. Ale co když bude v tom vstupním polí ($configurationData)

Reply:
doplnil jsem